types/ckeditor__ckeditor5-word-count/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es6",
```

Comment:
why are you removing this?

Reply:
thanks! Whenever someone edits default config, it adds special label for maintainers.